tests/foreman/ui/test_reporttemplates.py

```diff
@@ -27,8 +27,12 @@
 from lxml import etree
 from nailgun import entities
 
+from robottelo.cli.factory import setup_org_for_a_custom_repo
```

Comment:
Use it from `target_sat.cli_factory.setup_org_for_a_custom_repo` instead of importing it explicitly!